docs/src/development/email.md

```diff
@@ -105,28 +105,6 @@ Beware of the potential security problems when using the `mail()` function,
 which arise when using user-supplied input in the fifth (`$additional_parameters`) argument.
 See the [PHP `mail()` documentation](http://php.net/manual/en/function.mail.php) for more information.
 
-### SwiftMailer
```

Comment:
Swiftmailer is not maintained anymore and in any case, the docs below are only for Symfony, so not relevant anymore.